docs/sql-reference/sql-statements/data-definition/ALTER TABLE.md

```diff
@@ -307,6 +307,17 @@ Note:
 1. All columns in the index must be written.
 2. The value column is listed after the key column.
 
+#### Add a generated column
+
+Syntax:
+
+```sql
+ALTER TABLE [database.]table
+ADD col_name data_type [NULL] AS generation_expr [COMMENT 'string']
+```
+
+specifies the generated column and its expression. [The generated column](../generated_column.md) can be used to precompute and store the results of expressions, which significantly accelerates queries with the same complex expressions. Since v3.1, StarRocks supports generated columns.
```

Comment:
You can specify generated columns and their expressions. [Generated columns] can be used to xxx